middlewares/validators/tags.js

```diff
@@ -0,0 +1,30 @@
+const Joi = require("joi");
+const { TYPES } = require("../../constants/tags");
+
+const createTag = async (req, res, next) => {
```

Comment:
The function body and name is not matching,

In the function body, there is post-data validation, and names give hints like adding or creating tags,

better name can be  validAddTag or validTagBody,

Reply:
fixed it